Mathlib/Analysis/Normed/Ring/Lemmas.lean

```diff
@@ -214,3 +215,60 @@ instance Int.instNormOneClass : NormOneClass ℤ :=
 
 instance Int.instNormMulClass : NormMulClass ℤ :=
   ⟨fun a b ↦ by simp [← Int.norm_cast_real, abs_mul]⟩
+
+section NormMulClass
+
+variable [NormedRing α] [NormMulClass α] {a : α}
```

Comment:
Does everything still work if you make this `NonUnital`?

Reply:
It does indeed, well spotted.